service/passenger_service.py

```diff
@@ -0,0 +1,42 @@
+import textwrap
+
+from sqlalchemy import text, select
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from model.drive_order import DriveOrder, DRIVE_ORDER_TABLE
+
+SAVE_RATING_QUERY_TEMPLATE = textwrap.dedent(
+    f"""
+INSERT INTO {DRIVE_ORDER_TABLE} (user_id, passengers_amount, status, source_location, dest_location)
+VALUES (:user_id, :passengers_amount, "NEW", :source_location, :dest_location)
+"""
+)
+
+
+class PassengerService:
+    def __init__(self, session: AsyncSession):
+        self._session = session
+
+    async def save(self, order: DriveOrder) -> DriveOrder:
+        params = {"user_id": order.user_id, "passengers_amount": order.passengers_amount,
```

Comment:
use sqlalchemy's session save, example can be found in rating service

Reply:
i'm pretty sure its the same as in rating service, what am I missing?

Reply:
You should take the example from ImageService. there, we save an sqlalchemy's object instead of a dict.
Your use case is as simple(even simpler) as ImageService. 
rating service is a special use case.

Long version:
In rating service we need to perform a calculation on update so the save there is an insert-on-conflict-update kind of query.
Because we do not want to go through all the table once a drivers profile is retrieved, each time a user rates a driver, the driver's score is updated and this is why I used the text query instead of sqlalchemy object.